src/pyansys_sphinx_theme/__init__.py

```diff
@@ -39,3 +41,122 @@ def setup(app):
         "parallel_read_safe": True,
         "parallel_write_safe": True,
     }
+
+
+latex_elements = {
+    "preamble": r"""
+\makeatletter
+    \renewcommand{\sphinxmaketitle}{%
+        \newsavebox\imagebox
+        \newsavebox\myTitle
+        % native LaTeX command. Specifies a page with no formatting.
+        % Page number restarts after this page
+        \begin{titlepage}
+        % san serif
+        \sffamily
+        %
+        % set to 0 so that we don't have any blank space at the top of the page
+        \topskip0pt
+        %
+        % removes all margins
+        \advance\voffset -1in %
+        \advance\hoffset -1in %
+        \advance\vsize 2in %
+        \advance\hsize 2in %
+        %
+        % load the graphics into a LaTeX (not TeX) "savebox". We use this load
+        % to obtain the height later, and to insert it into the doc later
+        \sbox{\imagebox}{\includegraphics{drawing}}%
+        %
+        % put the picture in a zero sized box to get a "background effect"
+        % everything else will draw over it
+        \hbox to 0in {\vbox to 0in {\usebox{\imagebox}}}
+        %
+        % how it worked before savebox method
+        %\hbox to 0in {\vbox to 0in {\includegraphics{drawing}}} %
```

Comment:
Let us use a more explanatory name here. As a suggestion, consider:

```suggestion
        %\hbox to 0in {\vbox to 0in {\includegraphics{watermark}}} %
```

The same applies in other places of the string.